LPS-63470

hhuijser opened the pull request.

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/2300/">test-portal-acceptance-pullrequest(master)</a></p>

hhuijser commented:
Hey Mike,

I removed the warning for now, so it will at least pass SF.
I need to look into this more, but somehow everything was building fine for me, even when I removed the '-dsannotations-options: inherit' from the bnd.bnd. (I did 'gradlew clean' and 'gradlew install').

Any suggestions how I can test this?

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 40 minutes 37 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/be7f5ea07353ab575bef1c9269b851d015c8501f">be7f5ea07353ab575bef1c9269b851d015c8501f</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/2300/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26575/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18645/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk7/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18646/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/21890/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/1</a></li><li><strike><strong><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26577/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/10</a></strong></strike></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18929/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/11</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22117/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/12</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/17925/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/13</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/25516/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/14</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/25517/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/15</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18649/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/16</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19538/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/17</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/17927/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/18</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26811/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/19</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18647/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/2</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18215/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/20</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30134/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/21</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26812/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/22</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19539/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/23</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/17922/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/3</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30251/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/4</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/29844/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/5</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19648/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/6</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18927/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/7</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22116/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/8</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18928/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/9</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19540/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26813/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql56-jdk7</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30135/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18216/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/17828/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql56-jdk7</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26387/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk7</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26814/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/21891/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile-jdk7</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19459/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk7</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18930/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler-jdk7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/21892/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/21893/">test-portal-acceptance-pullrequest-batch(master)/tck-jdk7</a></li><li><strike><strong><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19460/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk7</a></strong></strike></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/2280/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2300/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/2300/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>38 Jobs Passed.<br />3 Jobs Failed.</p><pre>Completed with the status of FAILURE.</pre></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26577/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/10</a></h5><h6>Job Results:</h6><p>23 Tests Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/26577//testReport/com.liferay.poshi.runner/PoshiRunner/test_Calendar_ViewCalendarTimeZoneDefaults_">PoshiRunner.test[Calendar#ViewCalendarTimeZoneDefaults]</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/2300/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/functional-tomcat8-mysql56-jdk7/10">Poshi Report</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/2300/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/functional-tomcat8-mysql56-jdk7/10">Poshi Summary</a> - <a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/26577//console">Console Output</a></li></ol></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19460/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk7</a></h5><h6>Job Results:</h6><p>2226 Tests Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/19460//testReport/com.liferay.portal.modules/ModulesStructureTest/testScanBuildScripts">ModulesStructureTest.testScanBuildScripts</a></li></ol></li></ol>
